validator/sawtooth_validator/networking/handlers.py

```diff
@@ -58,6 +61,27 @@ class ConnectHandler(Handler):
     def __init__(self, network):
         self._network = network
 
+    @staticmethod
+    def is_valid_endpoint_host(interfaces, endpoint):
```

Comment:
Does this need to take an `interfaces` argument? It looks like there are two calls made to this function, and they both specify the same interfaces. Having `is_valid_endpoint_host` construct the list of interfaces itself would be simpler for callers. If it does need to be specified, maybe the signature could be `(endpoint, interfaces=None`), with a `None` arg defaulting to `["*", ".".join(["0", "0", "0", "0"])] + netifaces.interfaces()`.

(My motivation is that I'm rewriting validator `main` and that task would be easier with the interface stuff cut out.)

Reply:
I wouldn't worry about this.  It's irrelevant to this PR